src/formats/MMTF.cpp

```diff
@@ -301,6 +305,115 @@ void MMTFFormat::read(Frame& frame) {
     modelIndex_++;
 
     atomSkip_ = atomIndex_;
+
+    const auto original_size = frame.size();
+
```

Comment:
Might be nice to split up `MMTF::read()` in multiple sub-functions (`MMTF::read_atoms`, `MMTF::read_secondary`, `MMTF::read_symmetries` ...), it is starting to grow quite large =)

Reply:
I agree! I've refactored it, some of the comments maybe drowned out as a result.